src/styles/star.scss

```
@@ -0,0 +1,41 @@
+@import "./value.scss";
+
+.card__block-stars {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  margin: 0 0 24px;
+
+  .stars {
+    display: flex;
+  }
+
+  .stars__star {
+    background-repeat: no-repeat;
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-position: center;
+  }
```

Use SASS nesting in scss files, like you do in index.scss with card block
```suggestion
  .stars {
    display: flex;
    
    &__star {
      background-repeat: no-repeat;
      background-image: url(../images/star.svg);
      height: 16px;
      width: 16px;
      background-position: center;
    }
  }
```